packages/client/public/root-cookie-accessor-template.html

```diff
@@ -56,9 +56,14 @@
               break
             } else {
               const hasAccess = await document.hasStorageAccess()
-              const storageAccessPermission = await navigator.permissions.query({
-                name: 'storage-access'
-              })
+              let storageAccessPermission = { state: 'denied' } 
```

Comment:
I am just wondering why are we setting it as `denied`? Should it be set as denied in catch block? Otherwise use the response of `navigator.permissions.query` method?